.eslintrc

```diff
@@ -7,6 +7,9 @@
     "browser": true
   },
   "rules": {
+    "react/jsx-props-no-spreading": { "custom": "ignore" },
```

Comment:
What does this do?

Reply:
It doesn't allow prop spreading like `{...other}` in the code below. I set the rule so that it allows prop spreading in custom components. Also here are the [docs](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md) for more info

```
 <GatsbyLink
        className={classNames('usa-link', className)} 
        activeClassName={activeClassName}
        partiallyActive={partiallyActive}
        to={to}
        {...other}
      >
        {children}
      </GatsbyLink>
```